.github/workflows/build.yaml

```diff
@@ -184,12 +275,14 @@ jobs:
           ARGOCD_APPS_REPO: ${{ secrets.ARGOCD_APPS_REPO }}
 
       - name: Update k8s deployment
+        if: needs.set-environment.outputs.env-variable != 'unknown'
```

Comment:
this was set on a whole action level for `update-deployment-tags:` why do we need to have this condition again?